client/src/models/trees.js

```diff
@@ -9,118 +8,147 @@ const trees = {
+    rowsPerPage: 25,
     order: 'asc',
     orderBy: 'id',
     allIds: [],
     byId: {},
     displayDrawer: {
-      'isOpen': false
+      isOpen: false
     }
   },
   reducers: {
     selectAll(state) {
-      return { ...state }
+      return { ...state };
     },
     getTree(state, tree) {
       return { ...state, tree };
     },
-    getTrees(state, payload, { page, rowsPerPage, order, orderBy }) {
-      return { ...state, data: payload, page: page, rowsPerPage: rowsPerPage, order: order, orderBy: orderBy };
+    getTrees(state, payload, { rowsPerPage, order, orderBy }) {
+      return {
+        ...state,
+        data: payload,
+        rowsPerPage: rowsPerPage,
+        order: order,
+        orderBy: orderBy
+      };
     },
     receiveTreeCount(state, payload) {
-      return { ...state, treeCount: payload.count }
+      return { ...state, treeCount: payload.count };
     },
     receiveLocation(state, payload, { id, address }) {
       if (address === 'cached') {
-        return state
+        return state;
       } else {
-        const byId = Object.assign({}, state.byId)
-        if (byId[id] == null) byId[id] = {}
-        byId[id].location = payload.address
-        return { ...state, byId }
+        const byId = Object.assign({}, state.byId);
+        if (byId[id] == null) byId[id] = {};
+        byId[id].location = payload.address;
+        return { ...state, byId };
       }
     },
     // TODO: not quite sure if we need to keep this. I'll leave it until merge
     receiveStatus(state, payload) {
-      return { ...state, status: payload }
+      return { ...state, status: payload };
     },
     toggleDisplayDrawer(state) {
-      return { displayDrawer: { isOpen: !state.isOpen } }
+      return { displayDrawer: { isOpen: !state.isOpen } };
     },
     openDisplayDrawer(state) {
-      return { displayDrawer: { isOpen: true } }
+      return { displayDrawer: { isOpen: true } };
     },
     closeDisplayDrawer(state) {
-      return { displayDrawer: { isOpen: false } }
+      return { displayDrawer: { isOpen: false } };
     }
   },
   effects: {
-    async getTreesWithImagesAsync({ page, rowsPerPage, orderBy = 'id', order = 'asc' }) {
-      const query = `${API_ROOT}/trees?filter[order]=${orderBy} ${order}&filter[limit]=${rowsPerPage}&filter[skip]=${page * rowsPerPage}&filter[fields][imageUrl]=true&filter[fields][lat]=true&filter[fields][lon]=true&filter[fields][id]=true&filter[fields][timeCreated]=true&filter[fields][timeUpdated]=true&filter[where][active]=true&field[imageURL]`;
-      Axios.get(query)
-        .then((response) => {
-          this.getTrees(response.data, { page: page, rowsPerPage: rowsPerPage, orderBy: orderBy, order: order });
-        })
+    async getTreesWithImagesAsync({
+      page,
+      rowsPerPage,
+      orderBy = 'id',
+      order = 'asc'
+    }) {
+      const query = `${API_ROOT}/trees?filter[order]=${orderBy} ${order}&filter[limit]=${rowsPerPage}&filter[skip]=${page *
```

Comment:
A method to make this query string might be good (I realize these are mostly formatting changes though on your part)